contracts/mocks/ERC20CloneFactory.sol

```diff
@@ -19,15 +19,12 @@ contract ERC20CloneFactory is CloneFactory {
 
   address public masterAddress;
 
-  event CloneCreated(address cloneAddress);
-
-  constructor(address _masterAddress) public {
-    masterAddress = _masterAddress;
+  constructor() public {
+    masterAddress = new ERC20Mock(msg.sender, 0);
```

Comment:
Consider adding a comment here, stating that this constructor will only run in the `master` context, and not for all other created proxies.